### PR TITLE
Added B+E page into office

### PR DESCRIPTION
**Pull request by edwardwilson**
## Description
Refactored office page for cat b+e
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/office/cat-be/office.cat-be.page.ts [hunk 1]
```diff
@@ -0,0 +1,661 @@
...
+  seriousFaults$: Observable<(MultiFaultAssignableCompetency | CommentedCompetency)[]>;
+  isRekey$: Observable<boolean>;
+}
+
+@IonicPage()
+@Component({
+  selector: 'page-office',
+  templateUrl: 'office.cat-be.page.html',
+})
+export class OfficeCatBEPage extends PracticeableBasePageComponent {
```
Comment (rossfauldsbjss) [inline src/pages/office/cat-be/office.cat-be.page.ts]: Cat BE pages should be BasePageComponent not PracticeableBasePageComponent. Just needs a couple of things removed, including practice header in the html.

File: src/pages/office/cat-b/__tests__/office.cat-b.spec.ts [hunk 2]
```diff
@@ -15,68 +15,68 @@ import {
   PlatformMock,
```
Comment (TomBillingtonUK) [inline src/pages/office/cat-b/__tests__/office.cat-b.spec.ts]: missing page in the file name

**Review by ammarhaiderbjss — APPROVED** [review]
Looks good provided everything compiles!

File: src/pages/office/cat-b/office.cat-b.page.ts [hunk 3]
```diff
@@ -170,9 +170,9 @@ interface OfficePageState {
 @IonicPage()
 @Component({
   selector: 'page-office',
-  templateUrl: 'office.html',
+  templateUrl: 'office.cat-b.page.html',
```
Comment (TomBillingtonUK) [inline src/pages/office/cat-b/office.cat-b.page.ts:173]: you've not updated the selector in the scss file for the page

File: src/pages/office/cat-be/office.cat-be.page.ts [hunk 4]
```diff
@@ -0,0 +1,661 @@
...
+  displayDrivingFaultComments$: Observable<boolean>;
+  weatherConditions$: Observable<WeatherConditions[]>;
+  dangerousFaults$: Observable<(MultiFaultAssignableCompetency | CommentedCompetency)[]>;
+  seriousFaults$: Observable<(MultiFaultAssignableCompetency | CommentedCompetency)[]>;
+  isRekey$: Observable<boolean>;
+}
+
+@IonicPage()
+@Component({
+  selector: 'page-office',
```
Comment (TomBillingtonUK) [inline src/pages/office/cat-be/office.cat-be.page.ts]: this selector needs to be updated and scss file updated to match the selector

**Review by rossfauldsbjss — APPROVED** [review]
Looks good, assuming Tom's  comments are followed.

**Review by TomBillingtonUK — APPROVED** [review]
Looks good, a couple of things which you've missed and need to be changed

**Pull request merged**